scripts/submit_dpo_job.py

```diff
@@ -183,15 +183,21 @@ def parse_args(args):
             env['value'] = False
         if env['name'] == "WANDB_PROJECT":
             env['value'] = wandb_project
-    # d['tasks'][0]['envVars'].append({
-    #     'name': 'WANDB_API_KEY', 'value': wandb_api_key
-    # })
     d['tasks'][0]['envVars'].append({
         'name': 'WANDB_NAME', 'value': exp_name
     })
     d['tasks'][0]['envVars'].append({
         'name': 'WANDB_RUN_GROUP', 'value': experiment_group
     })
+    d['tasks'][0]['envVars'].append({
+        'name': 'BEAKER_TOKEN', 'secret': f"{getpass.getuser()}_BEAKER_TOKEN"
```

Comment:
where does getpass get the username from? I ask since my username is different depending on what machine im using....